### PR TITLE
[3.2] Add information about log levels [docs]

### LINKED ISSUE #209: Document framework for consistent logging level classification
Presently, we lack a consistent framework for logging level classification. We need to document and publish this in a destination determined by @randomnobody

### PR DESCRIPTION
**Pull request by heifner**
Provide information about logging levels and their expected use cases.

Resolves #209

### CODE REVIEW

File: docs/01_nodeos/06_logging/01_logging-levels.md [hunk 1]
```diff
@@ -64,3 +64,43 @@ Sample `logging.json`:
  ]
 }
 ```
+
+## Expected output of log levels
+
+* `error`
+  * Log output that likely requires operator intervention. Error level logging should 
+    be reserved for conditions that are completely unexpected or otherwise needs human
```
Comment (linh2931) [inline docs/01_nodeos/06_logging/01_logging-levels.md:72]: or an indication of a software error like impossible value of an enum, out of boundary array access, null pointers... something pointing to a bug

Comment (heifner) [reply docs/01_nodeos/06_logging/01_logging-levels.md:72]: Thanks. Incorporated this into the description.

**Review by linh2931 — APPROVED** [review]

**Review by larryk85 — APPROVED** [review]

**Comment by heifner** [comment]
Ready for DevRel

**Review by lparisc — APPROVED** [review]
Just made a few minor edits - see commits.

**Pull request merged**